web/src/test/java/com/epam/rft/atsy/cucumber/settings/positions/PositionsStepDefs.java

```diff
@@ -22,118 +24,228 @@
 import cucumber.api.java.en.When;
 
 public class PositionsStepDefs {
+  private DriverProvider driverProvider;
   private String positionsListContents;
   private String positionName;
 
-  @And("^the user is on an Atsy page$")
-  public void the_user_is_on_an_Atsy_page() throws Throwable {
-    assertThat(getDriver().getCurrentUrl(), containsString("secure"));
+  public PositionsStepDefs(DriverProvider driverProvider) {
```

Comment:
There is no need to pass the `DriverProvider` instance to this class.

Please remove this constructor and the field above.